src/wp-admin/js/dashboard.js

```diff
@@ -91,8 +151,17 @@ jQuery(document).ready( function($) {
 			}
 		} );
 
+		// Change the QuickPost action to the publish value.
 		$('#publish').click( function() { act.val( 'post-quickpress-publish' ); } );
 
+		/**
+		 * Adds accessibility context to inputs
+		 *
+		 * Use the 'screen-reader-text' class to hide the label when entering a value.
```

Comment:
Can you clarify a bit more what you mean? We don't completely understand the relation between screen-reader-text and ```#title, #tags-input, #content```. We don't really see how/where the screen-reader-text is used.